l10n_it_riba/models/riba_config.py

```diff
@@ -24,17 +25,20 @@ class RibaConfiguration(models.Model):
         "res.partner.bank",
         "Bank Account",
         required=True,
+        check_company=True,
```

Comment:
> Per coerenza penso devi aggiungere il dominio sulla company anche per il campo `bank_id` Non ricordo nelle altre versioni se avevamo fatto diversamente e se c'è un motivo, però è l'unico campo che non ha questo dominio e se si spuntano tutte le aziende vengono mostrate tutti i conti bancari invece dei soli dell'azienda selezionata

Grazie della revisione!
Qui viene aggiunto `check_company` al campo `bank_id`, quindi verrà iniettato il dominio sulla company come scrissi in https://github.com/OCA/l10n-italy/pull/3617#discussion_r1362344818:

> se il campo non avesse già un domain, verrebbe anche aggiunto automaticamente il domain per mostrare solo i record corretti, vedi in https://github.com/odoo/odoo/blob/c8cc82f24a913f633bb959045fd3767d8ab093d8/odoo/fields.py#L2525.

Non ho capito qual è la modifica che suggerisci, forse intendi un altro campo `bank_id`?

Reply:
In linea teorica il tuo ragionamento è corretto, ma nel dominio del campo compare solo questo che ho evidenziato senza riferimenti al campo `company_id`

![image](https://github.com/OCA/l10n-italy/assets/93990941/4f2d9235-b507-4062-bff5-929eae7f90f4)

In ogni caso ho notato che il campo `company_id` del modello `bank.account` è readonly e sempre vuoto perciò non servirebbe neanche un dominio che lo verifichi

![image](https://github.com/OCA/l10n-italy/assets/93990941/8979bf3b-6264-446d-aa25-22b3e5e94957)

Approvo la PR